LipidMatch_Flow_3.5/Background_Files_LipidMatch_Flow/LipidMatch_Flow/LipidMatch_Distribution/LipidMatch.r

```diff
@@ -1568,6 +1568,39 @@ remove_duplicates<-function(data,row_start,adduct_col,annotation_col,window,RT_c
   return(data)
 }
 
+genEIC <- function(path_to_EIC_gen_exe='../python_scripts/EICgen/dists/EICgen/EICgen.exe', # best practice might
+                   # be to define relative paths to executables at the top of the file, using the getwd() function
+                   mz_column = 6,

```

Comment:
# I redefined these variables according to how they are previously defined by the user in the R code. I have not tested the syntax for the directory portion, but I think you understand what I am getting at:
mz_column = 6,
rt_column = 7,
zoom_window = RT_Window*6,
mz_tolerance = PrecursorMassAccuracy,
feature_id_col = 11+CommentColumn,
target_file = 'NegIDed_FIN.csv',
#Getting directory for mzXML files by going up two levels from output directory, and then going into the Temp_Work folder
target_dir = paste(dirname(dirname(OutputDirectory)),"/Temp_Work/",sep="")

Reply:
Excellent! - I added a commit to include these in the parameter definitions, though I think you'll probably still want to pass them in when you call the function to be fastidious